src/pages/model/stackview/ModelPageStackViewContent.svelte

```diff
@@ -30,24 +30,27 @@
 <h1 class="sr-only">{$t('content.index.toolProcessCards.model.title')}</h1>
 <div class="flex flex-col h-full bg-backgrounddark">
   <div class="flex justify-center space-x-10 py-5 text-xl border-b-3 border-b-gray-200">
-    <Information
-      underlineIconText={false}
-      isLightTheme={false}
-      iconText={$t('content.model.output.estimatedGesture.iconTitle')}
-      titleText={$t('content.model.output.estimatedGesture.descriptionTitle')}
-      bodyText={$t('content.model.output.estimatedGesture.descriptionBody')} />
-    {#if $bestPrediction}
-      <p class="font-semibold text-2xl">
-        {$bestPrediction.name}
-      </p>
-      <p class="bg-secondary text-white rounded w-15 text-center">
-        {Math.round($bestPrediction.confidence.currentConfidence * 100)}%
-      </p>
-    {:else}
+    <div class="flex space-x-2 justify-center">
```

Comment:
```suggestion
    <div class="flex justify-end basis-1/2">
```

Reply:
I don't think the `space-x-2` is doing anything here?

Reply:
Oh cool, I like `basis-1/2`